qfieldsync/gui/package_dialog.py

```diff
@@ -140,20 +136,51 @@ def setup_gui(self):
         else:
             self.show_package_page()
 
-    def get_export_folder_from_dialog(self):
+    def get_export_folder(self) -> str:
         """Get the export folder according to the inputs in the selected"""
-        # manual
-        return self.manualDir.text()
+        export_dirname = self.qfield_preferences.value("exportDirectoryProject")
+        if not export_dirname:
+            export_dirname = os.path.join(
+                self.qfield_preferences.value("exportDirectory"),
+                fileparts(QgsProject.instance().fileName())[1],
+            )
+        return QDir.toNativeSeparators(str(export_dirname))
+
+    def get_export_filename_suggestion(self) -> str:
+        export_folder = Path(self.get_export_folder())
+        full_project_name_suggestion = export_folder.joinpath(
+            f"{self.project.baseName()}_qfield.qgs"
+        )
+        return str(full_project_name_suggestion)
+
+    def set_export_filename_suggested(self) -> None:
+        self.packagedProjectFileWidget.setFilePath(
+            self.get_export_filename_suggestion()
+        )
```

Comment:
I failed to find who uses this? In any case, such functions should not exist, it's better to do it explicitly what you do in the function body.

Reply:
You are completely right, the function is not needed!